test/test_crud.py

```diff
@@ -74,7 +75,12 @@ def run_playbook_vcr(tmpdir, module, extra_vars=None, record=False):
 
     cache_dir = tmpdir.join('cache')
     cache_dir.ensure(dir=True)
-    os.environ['FAM_TEST_APYPIE_CACHE_DIR'] = cache_dir.strpath
+    os.environ['XDG_CACHE_HOME'] = cache_dir.strpath
+    json_cache = cache_dir / 'apypie/https___foreman.example.com/v2/default.json'
```

Comment:
If we used the value from `vars/server.yml` to generate this path, we would use the same apidoc for recording as well as playing back. I think, that would spare us a lot of confusion.

Also we need a documented way to include a new default.json in case you are developing for a plugin.

Reply:
You are correct. I was somehow refraining from having to parse that yaml, but it doesn't really cost much.

Good point with the docs. Will add some to the new dev section in the readme.

Reply:
both done.